hip/CMakeLists.txt

```diff
@@ -179,19 +178,22 @@ set(GINKGO_HIP_SOURCES
     preconditioner/jacobi_kernels.hip.cpp
     preconditioner/jacobi_simple_apply_kernel.hip.cpp
     reorder/rcm_kernels.hip.cpp
-    solver/bicg_kernels.hip.cpp
-    solver/bicgstab_kernels.hip.cpp
-    solver/cg_kernels.hip.cpp
-    solver/cgs_kernels.hip.cpp
-    solver/fcg_kernels.hip.cpp
     solver/gmres_kernels.hip.cpp
     solver/cb_gmres_kernels.hip.cpp
     solver/idr_kernels.hip.cpp
-    solver/ir_kernels.hip.cpp
     solver/lower_trs_kernels.hip.cpp
     solver/upper_trs_kernels.hip.cpp
     stop/criterion_kernels.hip.cpp
-    stop/residual_norm_kernels.hip.cpp)
+    stop/residual_norm_kernels.hip.cpp
+    ../common/components/precision_conversion.cpp
```

Comment:
Will hip take all *.cpp file not only *.hip.cpp?

Reply:
Yes, `hipcc` doesn't care about the file extension